src/App.css

```diff
@@ -5,4 +5,7 @@
 
 .margin-top {
   margin: 1rem 0rem;
```

Comment:
Here you can remove rem it's still worked.

Reply:
okay